_scripts/pages/store/cart.js

```diff
@@ -107,7 +107,13 @@ Promise.all([jQuery]).then(([$]) => {
             const $state = $('select[name="state"]', form)
             const $statelabel = $('label[for="state"]', form)
 
-            if (notify) plausible('Store: Country Change') // value
+            if (notify) {
+                plausible('Store: Country Change', {
+                    props: {
+                        Country: value.toString()
```

Comment:
Is value not already a string? More so, wouldn't it make more sense to merge this with the payment process event so we just get one event of "Purchase" with amount and country?

Reply:
We don't currently have the country as a field on the checkout page, which I assume is why it is being tracked here.